portfolio/src/main/java/com/google/sps/data/Score.java

```diff
@@ -0,0 +1,62 @@
+package com.google.sps.data;
+
+import javax.servlet.http.HttpServletRequest;
+import com.google.appengine.api.datastore.Entity;
+
+/**
+ * A class for storing a Score.
+ */
+public class Score {
+    private String name;
+    private int score;
+    private int rank;
+
+    /**
+     * No argument constructor.
+     * Set fields with setter methods.
+     */
+    public Score() {
```

Comment:
It seems like the only reference to the Score constructor is on line 56. If that's the case, then maybe we omit this constructor and use the one defined on line 21.

Reply:
Thanks for the observation. Fixed and will keep in mind in the future.